english/interact/fork/fork.md

```diff
@@ -22,4 +22,6 @@ You pick up your fork and..
 
 Do you:
 
-[Follow Mr marshmallow] (follow/follow.md)
\ No newline at end of file
+[Follow Mr marshmallow] (follow/follow.md)
+
+[Go back to sleep?] (sleep/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.